roles/download/templates/kubeadm-images.yaml.j2

```diff
@@ -21,5 +21,5 @@ etcd:
 {% endif %}
 dns:
   type: CoreDNS
-  imageRepository: {{ coredns_image_repo | regex_replace('/coredns.*$','') }}
-  imageTag: {{ coredns_image_tag }}
+  imageRepository: k8s.gcr.io/coredns/coredns
+  imageTag: v1.8.0
```

Comment:
Should be taken from
https://github.com/iguazio/kubespray/blob/bbd0ade6f7ede7b3106097223fdb7bc35ff98c77/roles/igz_offline/defaults/main.yml#L12